shark-cli/src/main/java/shark/SharkCliCommand.kt

```diff
@@ -177,6 +177,7 @@ class SharkCliCommand : CliktCommand(
           .start()
           .also { it.waitFor(10, SECONDS) }
 
+      val output = process.inputStream.bufferedReader().readText()
```

Comment:
Please add a comment explaining why this should be a variable and cannot be inlined. Something like
`// On Windows, process doesn't always exit; call to readText() makes it finish, hence we're getting the output before checking exit value`

Reply:
Also, can you confirm that this behaves correctly in case of error?